tests/sparseml/pytorch/image_classification/test_integration_helper_functions.py

```diff
@@ -27,3 +27,4 @@ def test_integration_helper_functions():
     assert image_classification.create_model
     assert image_classification.create_dummy_input
     assert image_classification.export
+    assert image_classification.graph_optimizations is None
```

Comment:
@bfineran is that right? For image classification, we do not have any graph optimizations available?